website/common/script/content/appearance/backgrounds.js

```diff
@@ -480,6 +480,11 @@ let backgrounds = {
     },
   },
   incentiveBackgrounds: {
+    violet: {
```

Comment:
Arguably? Note that the set is labeled "Plain Background Set", which this is very much another of.